recipe/meta.yaml

```diff
@@ -19,7 +19,8 @@ requirements:
         - numpy x.x
         - cython
         - flann
-        - gcc 4.8*  # [osx]
+        - isl 0.12.*   # [osx]
```

Comment:
Add this to run time too to ensure it won't get updated when people install this with other software.